core/driver/sources/src/main/resources/reference.conf

```diff
@@ -301,15 +308,6 @@ probe {
       }
     }
 
-    // `driver.headless` defines if IDE should run in a headless mode (without a graphical user interface). If you want
-    // to use it, set to `true`. Specifically, it adds " headless" to the basic launch command ("idea").
-    // Note: if `headless` is set to `true`, it takes precedence over `driver.display`, which means that
-    // IDE will run in the headless mode (and not in a "native" or "xvfb" mode).
-    // Additionally, unless run in headless mode, the driver captures the screenshots of the screen periodically,
-    // so it can then be exposed on the CI for the user to inspect the IDE state in case of failures. If you run in the
-    // headless mode - then no screenshots will be captured.
-    headless = false
```

Comment:
This is a breaking change in general (albeit on a rather rarely used option)... TBD if we want to first deprecate (but maintain support) for `driver.headless` for some time before it's finally removed

Reply:
I talked with @lukaszwawrzyk about this and his suggestion was that we can just remove this whole feature and follow your suggestion from https://github.com/VirtusLab/ide-probe/issues/306#issuecomment-1295046843
Also, in my opinion, let's remove it and not keep some pointless parts of code/configs. It is not used anyway.

Reply:
Ok, +1